policyengine_canada/parameters/gov/provinces/mb/tax/income/credits/personal_tax_credit/age/teen_age.yaml

```diff
@@ -0,0 +1,11 @@
+description: Manitoba provides tax credit to teenagers who are born in and after this year.
+values:
+  2022-01-01: 2004
+metadata:
+  unit: year
+  label: Manitoba teenage tax credit 
```

Comment:
rewrite the labels to 
Manitoba {name of credit} {purpose of file}

f.e.
Manitoba personal tax credits age credit amount

Reply:
all yaml files